Sources/Sentry/Public/SentryOptions.h

```diff
@@ -328,6 +328,18 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL sendClientReports;
 
+/**
+ * When enabled, the SDK tracks when the application stops responding for a specific amount of
+ * time defined by the `anrTimeoutInterval` option.
+ */
+@property (nonatomic, assign) BOOL anrEnable;
```

Comment:
`h`: This doesn't align with the other feature flags. I think this should be `enableANR` or `enableANRTracking`.

Reply:
No problem, Im fine changing it. 
I copied the name android is using.